packages/serverless/src/backend_connector.js

```diff
@@ -105,7 +110,11 @@ exports.sendMetrics = function sendMetrics(metrics, callback) {
 };
 
 exports.sendSpans = function sendSpans(spans, callback) {
-  send('/traces', spans, false, callback);
+  try {
+    send('/traces', spans, false, callback);
```

Comment:
Just for my curiosity: why would `send` throw a synchronous exception? And if it potentially does, we need to handle that in all `sendXxx` methods or put the try-catch directly into `send`. But I actually think `send` should not throw an exception in the first place.

Reply:
Otel fails silently with probably some empty `catch`. I added try/catch in our code temporarily to be able to see these silent errors.